src/gpt_computer_agent/config/logging.py

```
@@ -0,0 +1,108 @@
+    
+    def emit(self, record):
+        from loguru import logger
+        
+        try:
+            level = logger.level(record.levelname).name
+        except ValueError:
+            level = record.levelno
+        
+        # Find caller from where the logging call was made
+        frame, depth = sys._getframe(6), 6
+        while frame and frame.f_code.co_filename == logging.__file__:
```

Ruff (0.13.1)

Undefined name `logging`

(F821)